src/components/calculator.js

```diff
@@ -1,46 +1,30 @@
 /* eslint-disable react/button-has-type */
+import PropTypes from 'prop-types';
 import { Component } from 'react';
+import BonusList from './ButtonList';
+import Display from './Display';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export default class Calculator extends Component {
```

Comment:
- [ ] There is an error when a number is divided by zero kindly fix it.

<img src='https://user-images.githubusercontent.com/33323077/130067848-67f71847-d758-4a62-aab8-bcfd0f193324.png' height='250px'> |